bundles/org.openhab.binding.touchwand/src/main/resources/ESH-INF/thing/dimmer.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="touchwand"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="dimmer">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"></bridge-type-ref>
+		</supported-bridge-type-refs>
+		<label>TouchWand Dimmer Unit</label>
+		<channels>
+			<channel id="brightness" typeId="brightness">
```

Comment:
You could use the system definition of a dimmer. Then, you don't need the channel-type definition below.
```suggestion
			<channel id="brightness" typeId="system.brightness">
```